tools/wt-mcp/README.md

```diff
@@ -0,0 +1,229 @@
+# WiredTiger MCP Server
+
+A Model Context Protocol (MCP) server that provides tools to interact with WiredTiger databases. This server exposes WiredTiger functionality through MCP tools, allowing easy analysis and interaction with WiredTiger files and databases.
+
+## Overview
+
+The WiredTiger MCP Server is designed to provide capabilities for analysing, inspecting, and interacting with WiredTiger databases through a standardised protocol. This enables integration with AI assistants, agents and other tools that support the Model Context Protocol.
+
+### What is MCP?
+
+The Model Context Protocol (MCP) is a standardised way for AI assistants to interact with external tools, resources, and environments. It enables AI models to:
+
+- Execute code in various environments
+- Access and manipulate files
+- Interact with databases and APIs
+- Retrieve relevant information from large codebases
+
+For more information, visit [modelcontextprotocol.io](https://modelcontextprotocol.io).
+
+## Installation
+
+### Prerequisites
+
+- Python 3.13
+- uv (a fast Python package installer and resolver from Astral - see [astral.sh/uv](https://astral.sh/uv) for installation instructions)
+- WiredTiger Python API
+- MCP Python SDK
+
+### Setup
+
+0. Install `uv` if you haven't already. `uv` is a fast Python package installer.
+   You can install it using pip:
+   ```bash
+   pip install uv
+   ```
+   Or using the standalone installer:
+   ```bash
+   # On macOS and Linux.
+   curl -LsSf https://astral.sh/uv/install.sh | sh
+   ```
+   Alternatively, follow the official installation instructions at [astral.sh/uv](https://astral.sh/uv).
+
+1. Create and activate a virtual environment using `uv` (recommended):
+   ```bash
+   uv venv .venv
+   source .venv/bin/activate
+   ```
+
+2. Sync the project dependencies:
+   ```bash
+   uv sync
+   ```
+
+3. Ensure the WiredTiger Python API is available:
+   - Set the `WT_BUILDDIR` environment variable to the path of your WiredTiger build directory.
+   - To maintain a persistent environment variable available to the script, create a `.env` file in the mcp-server root with the following content:
+     ```bash
+     WT_BUILDDIR=/path/to/wiredtiger/build
```

Comment:
I would suggest creating `WT_HOME` and then use `WT_HOME` throughout your doc. This would avoid the different `path/to/wiredtiger` or `~/wiredtiger`.